docker/bin/push2deis.sh

```diff
@@ -8,7 +8,22 @@
 
 set -ex
 
-deis pull "$DOCKER_REPOSITORY:$GIT_COMMIT" -a $DEIS_APPLICATION
+BIN_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source $BIN_DIR/set_git_env_vars.sh
+
+DEIS_BIN="${DEIS_BIN:-deis}"
+
+if [[ "$DEIS_BIN" == "deis2" ]]; then
+    if [[ "$DEMO_MODE" == "true" ]]; then
+        DOCKER_REPO="mozorg/bedrock_demo:${GIT_COMMIT}"
+    else
+        DOCKER_REPO="mozorg/bedrock:${BRANCH_NAME}-${GIT_COMMIT}"
+    fi
+else
+    DOCKER_REPO="$DOCKER_PRIVATE_REPO:${GIT_COMMIT}"
```

Comment:
So far we've only been using public quay.io and docker hub, as the main use cases we had for private docker registries disappeared, and in the new infra the public registries perform better as each node can pull the image directly from the public registry in parallel, without having to first wait for the complete image to be copied to a cluster-local registry.